docs/guides/documentation-style.stories.mdx

```diff
@@ -68,20 +69,56 @@ Embed `Stories` inline, share working code snippets, and use clear **✅ Do** /
 
 ### Use hierarchy
 
-| **✅ Do**                    | **❌ Don't**                  |
-| :--------------------------- | :---------------------------- |
-| **Design guidance**          | **Design guidance**           |
-| -> When to use               | -> When to use                |
-| -> Related components        | **Callbacks**                 |
-| -> Content guidelines        | **Related components**        |
-| -> Accessibility             | -> Accessibility              |
-| **Implementation guidance**  | **Controlled vs uncontroled** |
-| -> Props                     | **Responsiveness**            |
-| -> Callbacks                 | **Component composition**     |
-| -> Component composition     | -> Props                      |
-| -> Controlled vs uncontroled |                               |
+| **✅ Do**                     | **❌ Don't**                  |
+| :---------------------------- | :---------------------------- |
+| **Design guidance**           | **Design guidance**           |
+| -> When to use                | -> When to use                |
+| -> Related components         | **Callbacks**                 |
+| -> Content guidelines         | **Related components**        |
+| -> Accessibility              | -> Accessibility              |
+| **Implementation guidance**   | **Controlled vs uncontroled** |
+| -> Props                      | **Responsiveness**            |
+| -> Callbacks                  | **Component composition**     |
+| -> Component composition      | -> Props                      |
+| -> Controlled vs uncontrolled |                               |
 
 Clearly distinguish code-specific guidance from conceptual guidance so consumers
 can find what they're looking for without feeling overwhelmed.
 
 The template generated by `npm run generate` should be your starting point.
+
+## Component documentation
```

Comment:
New section on the _Writing documentation_ page. I wanted a way to communicate where to write docs depending on the content

<img width="540" alt="Screenshot 2025-02-08 at 5 45 24 PM" src="https://github.com/user-attachments/assets/a925ba71-8f18-4cf2-be38-52ab59594e18" />